play_motion:
  motions:
    reach_arm:
      joints: [arm_1_joint, arm_2_joint, arm_3_joint, arm_4_joint, arm_5_joint, arm_6_joint, arm_7_joint]
      points:
      - positions: [1.61, -0.93, -3.14, 1.83, -1.58, -0.53, 0.00]
        time_from_start: 0.0
    open_gripper:
      joints: [gripper_left_finger_joint, gripper_right_finger_joint]
      points:
      - positions: [0.04, 0.04]
        time_from_start: 0.0
    pre_navigation:
      joints: [torso_lift_joint]
      points:
      - positions: [0.15]
        time_from_start: 0.0
    post_navigation:
      joints: [torso_lift_joint, head_1_joint, head_2_joint]
      points:
      - positions: [0.25, 0.0, 0.0]
        time_from_start: 0.0
    look_left:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [0.5, 0.0]
        time_from_start: 0.0
    look_right:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [-0.5, 0.0]
        time_from_start: 0.0